easybuild/easyconfigs/s/StringTie/StringTie-2.2.1-GCC-11.2.0.eb

```diff
@@ -0,0 +1,46 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'MakeCp'
+
+name = 'StringTie'
+version = '2.2.1'
+
+homepage = 'https://ccb.jhu.edu/software/stringtie/'
+description = 'StringTie is a fast and highly efficient assembler of RNA-Seq alignments into potential transcripts'
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+
+source_urls = ['https://github.com/gpertea/%(namelower)s/releases/download/v%(version)s']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['ce4eec532bbbe39af462be844afa6395ab38fa3418ef5fc2431e2d6194129527']
+
+dependencies = [
+    ('bzip2', '1.0.8'),
+    ('libdeflate', '1.8'),
+    ('XZ', '5.2.5'),
+    ('zlib', '1.2.11'),
+    ('HTSlib', '1.14'),
+]
```

Comment:
the `prepDE.py` requires `python2`, so we should probably include that as a dependency (and add the appropriate versionsuffix)

Reply:
Should we use the bare or non-bare Python variant here?

Reply:
full, the bare dependency is used when python is only a build dependency (but not runtime)

Reply:
Thanks for the clarification, I will change this.